omnibus/config/software/habitat.rb

```diff
@@ -0,0 +1,38 @@
+name "habitat"
+default_version "master"
```

Comment:
I think you'd want to use `latest` here vs `master`. Once again, you're installing a pre-compiled binary and not installing from source.